<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScene</class>
 <widget class="QWidget" name="MainScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainScene {
	background-color: #FFF8EF;
}

#scrollArea {
	border: none;
}

QScrollBar:vertical {
	border-radius: 1px;
	width: 6px;
}

QScrollBar::handle:vertical {
	background: orange;
	border-radius: 4px;
	height: 12px;
}

QScrollBar::add-line:vertical {
    border: 2px solid grey;
	background: grey;
}

QScrollBar::sub-line:vertical {
    border: 2px solid grey;
	background: grey;
	height: 12px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="dictionaryBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
	color: orange;
	background-color: white;
    border: 2px solid lightgrey;
    border-radius: 16px;
    padding: 4px 1px 4px 12px;
    min-width: 6em;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::on {
	border-bottom-right-radius: 0;
	border-bottom-left-radius: 0;
}

QComboBox::down-arrow {
    image: url(/usr/share/icons/crystalsvg/16x16/actions/1downarrow.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}

QComboBox QAbstractItemView {
    border: 2px solid lightgrey;
    selection-background-color: lightgray;
	padding: 5px;
}</string>
       </property>
       <item>
        <property name="text">
         <string>English - English</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>English - Vietnamese</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vietnamese - English</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="SearchBox" name="searchBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="titleLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="title">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Random new words</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>622</width>
        <height>369</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#scrollContent {
	border: none;
	background-color: #FFF8EF;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="WordCardGroup" name="wordGroup" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchBox</class>
   <extends>QLineEdit</extends>
   <header>searchbox.h</header>
  </customwidget>
  <customwidget>
   <class>WordCardGroup</class>
   <extends>QWidget</extends>
   <header>WordCard/wordcardgroup.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
